src/App/components/Dropdown/index.js

```diff
@@ -2,7 +2,13 @@ import React, { useEffect, useState, Fragment } from "react";
 import PropTypes from "prop-types";
 import { dropdown } from "@src/scripts/main";
 
-const DropdownSelect = ({ disabled, errorMessage, fullWidth, position }) => {
+const DropdownSelect = ({
+	disabled,
+	errorMessage,
+	fullWidth,
+	position,
+	noRotation = false,
```

Comment:
Avoid double negative if you can, it is easier to read rotation = true than rotation is true if noRotation is false.